enclave/include/enclave_context.h

```diff
@@ -122,11 +122,12 @@ class EnclaveContext {
     //  }
     //}
 
-    bool get_client_key(uint8_t* key) {
-      if (client_key_is_set)
+    void get_client_key(uint8_t* key) {
+      if (client_key_is_set) {
           memcpy(key, client_key, CIPHER_KEY_SIZE);
```

Comment:
Can you fix the spacing on this line? I don't think you even changed this specific line but it seems to be indented with 4 spaces instead of 2?

Reply:
Done.